### PR TITLE
CW-729: Automated tests for Create Wallets flow

### PR DESCRIPTION
**Pull request by Blazebrain**
Issue Number (if Applicable): Fixes #

# Description

Please include a summary of the changes and which issue is fixed / feature is added. 

# Pull Request - Checklist  

- [ ] Initial Manual Tests Passed
- [ ] Double check modified code and verify it with the feature/task requirements
- [ ] Format code
- [ ] Look for code duplication
- [ ] Clear naming for variables and methods

### CODE REVIEW

File: integration_test/test_suites/create_wallet_flow_test.dart [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+        await commonTestFlows.createNewWalletFromWalletMenu(walletType);
+
+        await dashboardPageRobot.confirmWalletTypeIsDisplayedCorrectly(walletType);
+      }
+
+      // Go to the wallet menu, provides a visual confirmation that all the wallets were correctly restored
+      await commonTestFlows.switchToWalletMenuFromDashboardPage();
+
+      await Future.delayed(Duration(seconds: 5));
```
Comment (OmarHatem28) [inline integration_test/test_suites/create_wallet_flow_test.dart:52]: same comment as the previous PR, let's make it automated

**Review by OmarHatem28 — CHANGES_REQUESTED** [review]
need to migrate to wallet groups flow

File: integration_test/test_suites/create_wallet_flow_test.dart (hunk 1 above)
Comment (Blazebrain) [reply integration_test/test_suites/create_wallet_flow_test.dart:52]: Alright, would adjust here also

**Comment by Blazebrain** [comment]
Closing this PR now as it's changes has been add to the most recent Automated Testing PR

**Pull request closed without merging**